### PR TITLE
[EOC-39 ]  Dashboard db api

### PR DESCRIPTION
**Pull request by adamklepacz**

### CODE REVIEW

File: server/controllers/shoppingList.js [hunk 1]
```diff
@@ -28,7 +28,6 @@ const getAllShoppingLists = (req, resp) => {
 };
 
 const getShoppingListById = (req, resp) => {
-  console.log('shoppig list of id:');
   resp.status(200).send(`Shopping list by id: ${req.params.id}`);
```
Comment (alemikolo) [inline server/controllers/shoppingList.js]: unnecessary code...

File: server/models/shoppingList.model.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const ShoppingListSchema = new Schema(
+  {
+    name: { type: String, required: true },
```
Comment (marekrozmus) [inline server/models/shoppingList.model.js]: please sort

File: server/controllers/shoppingList.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  const shoppingList = new ShoppingList({
+    description,
+    name
+  });
+
+  shoppingList.save((err, doc) => {
+    err
+      ? resp.status(400).send(err.message)
+      : resp
+          .location(`/shoppin-list/${doc._id}`)
```
Comment (marekrozmus) [inline server/controllers/shoppingList.js]: 'g' is missing in 'shopping'

File: server/routes/item.js [hunk 4]
```diff
@@ -21,4 +21,6 @@ router.delete('/:id/delete', deleteItemById);
 // Update product by id
 router.patch('/:id/update', updateItem);
 
-module.exports = router;
+module.exports = app => {
```
Comment (marekrozmus) [inline server/routes/item.js]: remove braces

File: server/routes/authorization.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+const router = express.Router();
+const { authenticate, authenticateCallback } = require('../config/auth');
+const { logout, setUserAndSession } = require('../controllers/userAuth');
+
+router.get('/auth/google', authenticate);
+router.get('/auth/google/callback', authenticateCallback, setUserAndSession);
+router.post('/logout', logout);
+
+module.exports = app => {
```
Comment (marekrozmus) [inline server/routes/authorization.js]: remove braces

File: server/routes/items.js [hunk 6]
```diff
@@ -7,4 +7,6 @@ const { getAllItems } = require('../controllers/items');
 // Get all items
 router.get('/:type?', getAllItems);
 
-module.exports = router;
+module.exports = app => {
```
Comment (marekrozmus) [inline server/routes/items.js]: remove braces

File: server/routes/shoppingList.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+  createNewList,
+  getAllShoppingLists,
+  getShoppingListById
+} = require('../controllers/shoppingList');
+
+router.get('/', getAllShoppingLists);
+router.get('/:id', getShoppingListById);
+router.post('/new-list', createNewList);
+
+module.exports = app => {
```
Comment (marekrozmus) [inline server/routes/shoppingList.js]: remove braces

**Pull request merged**